src/opm/parser/eclipse/EclipseState/Aquifer/Aquancon.cpp

```diff
@@ -50,26 +52,30 @@ namespace Opm {
             }
         }
 
-        void add_cell(std::unordered_map<std::size_t, Aquancon::AquancCell>& work,
+        void add_cell(const KeywordLocation& location,
+                      std::unordered_map<std::size_t, Aquancon::AquancCell>& work,
                       const EclipseGrid& grid,
                       int aquiferID,
-                      std::size_t global_index,
+                      std::size_t i, std::size_t j, std::size_t k,
```

Comment:
I don't think that's a good idea.  You're making the interface bigger only for the sake of generating an error message in an unlikely case.  I'd prefer to compute the location on the fly (`EclipseGrid::getIJK()`) if it's actually needed.

Reply:
Well - the "interface" is in an anonymous namespace - so it will not trickle very far; but if you still prefer that I will change to use `EclipseGrid::getIJK()`

Reply:
> he "interface" is in an anonymous namespace - so it will not trickle very far;

Be that as it may, you're still passing 24 bytes instead of 8 (on 64-bit Linux) into the function.

> if you still prefer that I will change to use `EclipseGrid::getIJK()`

I think I do, actually.

Reply:
Done!

Reply:
> Done!

Much appreciated.